src/components/views/settings/CrossSigningPanel.js

```diff
@@ -128,7 +128,6 @@ export default class CrossSigningPanel extends React.PureComponent {
         }
 
         const enabled = (
-            crossSigningPublicKeysOnDevice &&
```

Comment:
Isn't this going to cause the `summarisedStatus` status to say it's enabled when it isn't?

Reply:
weirdly, `crossSigningPublicKeysOnDevice` seems to only be true if the _private_ keys are also on the device.  Even if the device doesn't have keys, the cross-signing can be initialised.

Reply:
Yeah, `crossSigningPublicKeysOnDevice` is whether the public key is trusted or not. My point was that in the if block just below, we'll now only hit the second case if `crossSigningPrivateKeysInStorage` is true but `secretStorageKeyInAccount` is false. So this corrects the second if block but breaks the first?